usaspending_api/settings.py

```diff
@@ -198,49 +198,24 @@
     'version': 1,
     'disable_existing_loggers': False,
     'formatters': {
-        'specifics': {
-            '()': "pythonjsonlogger.jsonlogger.JsonFormatter",
-            'format': "%(asctime)s %(filename)s %(funcName)s %(levelname)s %(lineno)s %(module)s %(message)s %(name)s %(pathname)s"
-        },
-        'json': {
-            '()': "pythonjsonlogger.jsonlogger.JsonFormatter",
-        },
         'simpletime': {
             'format': "%(asctime)s - %(message)s",
             'datefmt': "%H:%M:%S"
+        },
+        'user_readable': {
+            '()': "pythonjsonlogger.jsonlogger.JsonFormatter",
+            'format': "%(timestamp)s %(status)s %(method)s %(path)s %(status_code)s %(remote_addr)s %(host)s " +
+                      "%(response_ms)d %(message)s %(request)s %(traceback)s"
         }
     },
     'handlers': {
-        'file': {
-            'level': 'DEBUG',
-            'class': 'logging.handlers.RotatingFileHandler',
-            'filename': os.path.join(BASE_DIR, 'usaspending_api/logs/debug.log'),
-            'maxBytes': 1024*1024*2,  # 2 MB
-            'backupCount': 5
-        },
-        'console_file': {
-            'level': 'INFO',
-            'class': 'logging.handlers.RotatingFileHandler',
-            'filename': os.path.join(BASE_DIR, 'usaspending_api/logs/console.log'),
-            'maxBytes': 1024*1024*2,  # 2 MB
-            'backupCount': 5,
-            'formatter': 'specifics'
-        },
-        'events_file': {
+        'server': {
             'level': 'INFO',
             'class': 'logging.handlers.RotatingFileHandler',
-            'filename': os.path.join(BASE_DIR, 'usaspending_api/logs/events.log'),
-            'maxBytes': 1024*1024*2,  # 2 MB
+            'filename': os.path.join(BASE_DIR, 'usaspending_api/logs/server.log'),
+            'maxBytes': 1024*1024*20,  # 20 MB
```

Comment:
surprised this isn't flagged as PEP8 issue

Reply:
This value might be adjusted later. Makes it easier to update, but I can change it if you have a suggestion.